SU-SWS/saa_victoria_subtheme
# ADAPT-476: Rebuilt and committed saa-logo in dist folder.

# READY FOR REVIEW

# Summary
- rebuilt to commit dist file for saa-logo

# Review By (Date)
- 7/17

# Urgency
- needed to reference file

# Steps to Test

1. pull down branch
1. run npm run publish, and there shouldn't be anything to commit / different.
2. we maybe should change the twig template that referenes this file in its source location to instead reference this file in the dist location?? cc @rebeccahongsf 

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- JIRA ticket 476 testing
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
